modules/Admin/views/components/tablerow.blade.php

```diff
@@ -0,0 +1,68 @@
+<tr>
+    <td>{{$user->id}}</td>
+    <td>{{$user->name}}</td>
+    @switch($user->organization)
+    @case('0')
```

Comment:
can we add indentation here?

Reply:
When I use the auto format feature: shift + alt + f, that is how it gets indented. I can change it manually but if I use auto format again then it will go back to that.

Reply:
Then keep it as it is